source/language/openpulse.rst

```diff
@@ -0,0 +1,650 @@
+OpenPulse Grammar
+=================
+
+OpenQASM allows users to provide the target system's implementation of quantum operations
+with ``defcal`` blocks . Calibration grammars are open to extension for system implementors. In
+this document, we outline one such grammar, OpenPulse, which maybe selected within a supporting
+compiler through the declaration defcalgrammar "openpulse" <VERSION>.
+
+This grammar is motivated by the original OpenPulse specification, a JSON wire-format for
+pulse-level quantum programs defined in the paper `Qiskit Backend Specifications for OpenQASM
+and OpenPulse Experiment`_:cite:`mckay2018`. The textual format described
```

Comment:
Prior to final acceptance, we should check for other work that ought to be cited.

Reply:
This might be worth adding? https://arxiv.org/pdf/2004.06755.pdf

Reply:
Addressed in upcoming commit 3877695